lib/action_subscriber/configuration.rb

```diff
@@ -8,13 +8,18 @@ class Configuration
                   :host,
                   :hosts,
                   :mode,
+                  :pass,
                   :pop_interval,
                   :port,
                   :prefetch,
                   :publisher_confirms,
+                  :user,
+                  :scheme,
+                  :ssl,
```

Comment:
If we get rid of that then we can't explode the parse response into settings like @mmmries wanted to.
```
settings = ::AMQ::Settings.parse_amqp_url(url)
settings.each do |key, value|
       send("#{key}=", value)
end
```